src/test/java/io/github/sskorol/testcases/AmazonTests.java

```diff
@@ -79,4 +80,35 @@ public void shouldSearchForLego(final Lego lego, final Account account) {
         customAssertThat(lego)
                 .hasPurchaseStatus(at(ProductPage.class).getPurchaseStatus());
     }
+
+    @Data(source = "shoes.json", entity = Shoes.class)
+    @Data(source = "accountAmazon.json", entity = Account.class)
+    @Test(dataProvider = "getDataCollection", dataProviderClass = DataSuppliers.class)
```

Comment:
Could you please add the following;
 `description` = "Name of your test"

Reply:
Will fix that comment with my PR.